src/runtime/handle.ts

```diff
@@ -138,6 +138,17 @@ export abstract class Handle {
   protected generateKey(): string {
     return this.idGenerator.newChildId(Id.fromString(this._id), 'key').toString();
   }
+
+  disable(particle?: Particle, storageProxy?: StorageProxy): void {
+    if (storageProxy != undefined) {
+      storageProxy.deregister(particle, this);
+    }
+    this.storage = StorageProxy.newDummyProxy(Type.fromLiteral({tag: 'Dummy'}));
```

Comment:
Either here, or where we define the dummy proxy, I thought it could be worth a comment explaining why we think having/using this is important. A brief note on desire to prevent potentially spurious async post-disable completion from raising errors/etc. So that a reader wondering why we don't just null it out has an answer.

Reply:
Added the comment!